python-frontend/typeshed_serializer/serializer/typeshed_serializer.py

```diff
@@ -136,7 +136,7 @@ def save_merged_symbols(is_debug=False, is_third_parties=False):
 
 def main():
     save_merged_symbols()
-    save_merged_symbols(is_third_parties=True)
```

Comment:
I guess this change is not needed.

Reply:
Fixed in 90ef511687a014c251b8a76215d7303933287b15